common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1456,14 +1457,26 @@ object PageElement {
     doc.getElementsByTag("iframe").asScala.headOption.map(_.attr("src"))
   }
 
-  private[this] def getIframeWidth(html: String): Option[Int] = {
+  private[this] def getIframeWidth(html: String, fallback: Int = 0): Option[Int] = {
     val doc = Jsoup.parseBodyFragment(html)
-    doc.getElementsByTag("iframe").asScala.headOption.map(_.attr("width").toInt)
+
+    doc
+      .getElementsByTag("iframe")
+      .asScala
+      .headOption
+      .map(_.attr("width"))
+      .map(attr => Try(attr.toInt).getOrElse(fallback))
```

Comment:
Nice, very clever use of a parameter here. Never thought of passing in a fallback. Elegant simplicity!